ansible/playbooks/webhooks/secrets.json.j2

```diff
@@ -0,0 +1,3 @@
+{
+    "website_update_key": "{{ ql_webhooks.website_update_key }}"
```

Comment:
Double checking: can you confirm if `ql_webhooks.website_update_key` is already in the vault (I'm not seeing the sekrets file touched in this PR)?

Reply:
I have double-checked this, i committed it at b5fb6d4